]--- a/src/OVAL/probes/unix/linux/systemdshared.h
+++ b/src/OVAL/probes/unix/linux/systemdshared.h
@@ -338,6 +338,8 @@ static DBusConnection *connect_dbus()
 		setenv("DBUS_SYSTEM_BUS_ADDRESS", dbus_address, 0);
 		/* We won't overwrite DBUS_SYSTEM_BUS_ADDRESS so that
 		 * user could have a way to define some non-standard system bus socket location */
+	} else {
+		setenv("DBUS_SYSTEM_BUS_ADDRESS", "unix:path=/run/dbus/system_bus_socket", 0);
 	}
 
 	conn = dbus_bus_get(DBUS_BUS_SYSTEM, &err);
